app/services/auto-approval/auto-approval-process.js

```diff
@@ -1,69 +1,71 @@
-const config = require('../../../config')
-
 const getDataForAutoApprovalChecks = require('../data/get-data-for-auto-approval-check')
+const getAutoApprovalConfig = require('../data/get-auto-approval-config')
 const insertClaimEventData = require('../data/insert-claim-event-data')
 const generateFailureReasonString = require('../notify/helpers/generate-failure-reason-string')
 const autoApproveClaim = require('../data/auto-approve-claim')
 const claimTypeEnum = require('../../constants/claim-type-enum')
 const statusEnum = require('../../constants/status-enum')
+const autoApprovalRulesEnum = require('../../constants/auto-approval-rules-enum')
+
+var autoApprovalChecks = {}
 
-const autoApprovalChecks = [
-  require('./checks/are-children-under-18'),
-  require('./checks/cost-and-variance-equal-or-less-than-first-time-claim'),
-  require('./checks/do-expenses-match-first-time-claim'),
-  require('./checks/has-claimed-less-than-max-times-this-year'),
-  require('./checks/has-uploaded-prison-visit-confirmation-and-receipts'),
-  require('./checks/is-claim-submitted-within-time-limit'),
-  require('./checks/is-claim-total-under-limit'),
-  require('./checks/is-latest-manual-claim-approved'),
-  require('./checks/is-no-previous-pending-claim'),
-  require('./checks/is-prison-not-in-guernsey-jersey'),
-  require('./checks/is-visit-in-past'),
-  require('./checks/visit-date-different-to-previous-claims')
-]
+autoApprovalRulesEnum.forEach(function (check) {
+  autoApprovalChecks[check] = require(`./checks/${check}`)
+})
 
 module.exports = function (reference, eligibilityId, claimId) {
-  var autoApprovalEnabled = config.AUTO_APPROVAL_ENABLED === 'true'
+  return getAutoApprovalConfig()
+    .then(function (config) {
+      if (config.AutoApprovalEnabled) {
+        return getDataForAutoApprovalChecks(reference, eligibilityId, claimId)
+          .then(function (autoApprovalData) {
+            var result = {checks: []}
 
-  if (autoApprovalEnabled) {
-    return getDataForAutoApprovalChecks(reference, eligibilityId, claimId)
-      .then(function (autoApprovalData) {
-        var result = {checks: []}
+            // Fail auto-approval check if First time claim, advance claim or status is PENDING
+            if (autoApprovalData.Claim.ClaimType === claimTypeEnum.FIRST_TIME ||
+              autoApprovalData.Claim.IsAdvanceClaim ||
+              autoApprovalData.Claim.Status === statusEnum.PENDING) {
```

Comment:
split into function that returns true/false if claim should be checked (removing need to comment)